index.html

```diff
@@ -4,12 +4,580 @@
+                            <figcaption class="tp-product-caption">
+                                <h3 class="tp-product-title">Biztar</h3>
+                                <p class="tp-product-price">$12.00</p>
+                                <p class="tp-prodcut-category">Business Consultant</p>
+                                <p class="tp-product-sub">UI Kits</p>
+                            </figcaption>
+                        </figure>
+                    </a>
+                </li>
+                <li class="tp-list-item">
+                    <a href="#" class="tp-product">
+                        <figure class="tp-product-figure">
+                            <img src="#" alt="photo" class="tp-product-image">
+                            <img src="#" alt="trophy icon" class="tp-icon">
+                            <figcaption class="tp-product-caption">
+                                <h3 class="tp-product-title">Auxell</h3>
+                                <p class="tp-product-price">$8.00</p>
+                                <p class="tp-prodcut-category">Company Proposal</p>
+                                <p class="tp-product-sub">Print Template</p>
+                            </figcaption>
+                        </figure>
+                    </a>
+                </li>
+                <li class="tp-list-item">
+                    <a href="#" class="tp-product">
+                        <figure class="tp-product-figure">
+                            <img src="#" alt="photo" class="tp-product-image">
+                            <img src="#" alt="trophy icon" class="tp-icon">
+                            <figcaption class="tp-product-caption">
+                                <h3 class="tp-product-title">Phitaya</h3>
+                                <p class="tp-product-price">$10.00</p>
+                                <p class="tp-prodcut-category">Handwritten Font</p>
+                                <p class="tp-product-sub">Fonts</p>
+                            </figcaption>
+                        </figure>
+                    </a>
+                </li>
+                <li class="tp-list-item">
+                    <a href="#" class="tp-product">
+                        <figure class="tp-product-figure">
+                            <img src="#" alt="photo" class="tp-product-image">
+                            <img src="#" alt="trophy icon" class="tp-icon">
+                            <figcaption class="tp-product-caption">
+                                <h3 class="tp-product-title">Rachelle</h3>
+                                <p class="tp-product-price">$16.00</p>
+                                <p class="tp-prodcut-category">Media Kit</p>
+                                <p class="tp-product-sub">Presentation</p>
+                            </figcaption>
+                        </figure>
+                    </a>
+                </li>
+            </ul>
+            <div class="tp-banner">

```

Comment:
Use the `<aside>` tag for content that is tangentially related to the main content. [Learn more about the `<aside>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/aside).